### ISSUE TITLE
Netlify-deployed docs site is down

### ISSUE THREAD

**Issue by kabaluyot** [dcmjs-org/dcmjs#389]

**Comment by pieper** [comment]
@wayfarer3130 looks like you and @dannyrb set this up - can you have a look?

**Comment by wayfarer3130** [comment]
I didn't setup any of this - not sure how it is configured or managed.

**Comment by pieper** [comment]
@wayfarer3130 do you know if @dannyrb is still available to give advice?

**Comment by wayfarer3130** [comment]
> @wayfarer3130 do you know if @dannyrb is still available to give advice?

No, I don't know if he is still available.  Let me ask Alireza if he can guide me on how to setup the netlify docs site.
